search_gov_crawler/elasticsearch/convert_html_i14y.py

```diff
@@ -0,0 +1,73 @@
+import os
+import hashlib
+import newspaper
+from datetime import datetime, timezone
+from urllib.parse import urlparse
+
+def convert_html(html_content: str, url: str, domain_name: str):
+    article = newspaper.article(url=url, input_html=html_content)
+    article.nlp()
+    description_en = article.meta_description or article.summary
+    content_en = article.text_cleaned or article.text or description_en or article.title
+
+    if content_en is None:
+        return None
+
+    time_now_str = current_utc_iso()
+    path = article.url or url
+
+    basename, extention = get_base_extension(url)
+    sha_id = generate_url_sha256(path)
+
+    i14y_doc = {
+        "audience": None,
```

Comment:
I wonder where we get his from currently.

Reply:
I think it's derived from the meta tags:
> <meta name="audience" ...> tag within the <head> section of the webpage's source code; the content within this tag will usually specify the target demographic or user group the website is designed for. 

It almost never there (especially for gov websites)

Reply:
Can we set it up even though it's almost never there?